simplq/src/components/pages/Home/home.module.scss

```diff
@@ -24,30 +24,7 @@
     margin: 2rem;
     font-size: larger;
   }
-  .create-join-form {
-    margin: auto; // auto for centering
-    padding-top: 2rem;
-    max-width: 35rem;
-    position: relative;
-    // z-index set greater than that of svg below,
-    // otherwise button becomes unclicable on wider screens
-    z-index: 3;
-    width: 100%;
-    .button-group {
-      @include center-horizontally();
-      flex-wrap: wrap;
-      button {
-        margin: 0.5rem 2rem;
-        min-width: 10rem;
-      }
-    }
-    .input-box {
-      min-height: 5rem;
-    }
-  }
   svg {
-    // z-index set to lover value than upper create-join-form
-    z-index: 2;
     // global margin of 1 rem set. Setting negative margin to fill till the edge
```

Comment:
Looks like it works fine without z-index now, let me know if you notice any problems anywhere.